### PR TITLE
feat: add last update and number of subscribers to store sort

### LINKED ISSUE #45: Sort Crops by number of subscribers and last update date

### PR DESCRIPTION
**Pull request by aveuiller**
Closes #45

### CODE REVIEW

File: bee/src/main/java/com/apisense/bee/ui/adapter/sort/Sorter.java [hunk 1]
```diff
@@ -1,4 +1,6 @@
-package com.apisense.bee.ui.adapter;
+package com.apisense.bee.ui.adapter.sort;
+
+import com.apisense.bee.ui.adapter.sort.SortComparator;
```
Comment (rsommerard) [inline bee/src/main/java/com/apisense/bee/ui/adapter/sort/Sorter.java]: This import is not required.

**Comment by aveuiller** [comment]
I had in mind that an image would be more entertaining, any thought about the design @rouvoy ?

**Comment by rsommerard** [comment]
I reconsidered my point of view, I think putting textual information can be annoying for the user. We can keep this presentation. :)

**Pull request merged**